CONTRIBUTING.md

```diff
@@ -24,23 +24,12 @@ This section describes how to build Contour from source.
 
 Contour uses [`go modules`][2] for dependency management.
 
-```
-go get github.com/projectcontour/contour
-```
-
-Go is very particular when it comes to the location of the source code in your `$GOPATH`.
-The easiest way to make the `go` tool happy is to rename Contour's remote location to something else, and substitute your fork for `origin`.
-For example, to set `origin` to your fork, run this command substituting your GitHub username where appropriate.
+Fork the repo and create a local clone for development.
```

Comment:
maybe split this into two steps for clarity for folks who are less familiar with this process:

1. Fork the repo (is there a link we can provide?)
2. Create a local clone (include the command below)

Reply:
Does this seem to be a viable link for fork: https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#fork-an-example-repository ?
I don't think they are URLs available to directly fork a repo.

Reply:
Yeah I think that's great.